src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,143 @@ JSON example:
+
+### (P)CASL-specific metadata fields
+These fields can only be used when `LabelingType` is `CASL` or `PCASL`.
+| **Key name**                  | **Requirement level** | **Data type** | **Description**                                                   |
+|-------------------------------|-----------------------|---------------|------------------------------------------------------------------------------------------|
+|  LabelingDuration                 | REQUIRED            | [number][] or [array][] of [numbers][]         | Total duration of the labeling pulse train, in seconds, corresponding to the temporal width of the labeling bolus for `(P)CASL`. Specify either one number for total time-series or provide an array of numbers of different values for each volume in case of sequential acquisitions with varying labeling duration. In the case that an array of numbers is provided, its length should be equal to the number of volumes specified in  `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_aslcontext.tsv`. An `m0scan` acquired within the ASL time-series, gets a value of zero. Corresponds to DICOM Tag 0018,9258 `ASL Pulse Train Duration`. |
+|  PCASLType                 | RECOMMENDED             | [string][]         |  Type the gradient pulses used in the `control` condition: `balanced` or `unbalanced`.    |
+|  CASLType                  | RECOMMENDED             | [string][]         |  Describes if a separate coil is used for labeling: `single-coil` or `double-coil`. |
+|  LabelingPulseAverageGradient                   | RECOMMENDED             | [number][]         |  The average labeling gradient, in milliteslas per meter. |
+|  LabelingPulseMaximumGradient                  | RECOMMENDED             | [number][]         | The maximum amplitude of the gradient switched on during the application of the labeling RF pulse(s), in milliteslas per meter. |
+|  LabelingPulseAverageB1                   | RECOMMENDED             | [number][]         | The average B1-field strength of the RF labeling pulses, in microteslas. As an alternative, `LabelingPulseFlipAngle` can be provided.  |
+|  LabelingPulseDuration                    | RECOMMENDED             | [number][]         | Duration of the individual labeling pulses, in seconds. |
+| LabelingPulseFlipAngle | RECOMMENDED  | [number][]         | The flip angle of a single labeling pulse, in degrees, which can be given as an alternative to `LabelingPulseAverageB1`. |
+|  LabelingPulseInterval                     | RECOMMENDED             | [number][]         | Delay between the peaks of the individual labeling pulses, in seconds. |
+
+### PASL-specific metadata fields
+These fields can only be used when `LabelingType` is `PASL`.
+| **Key name**                  | **Requirement level** | **Data type** | **Description**                                                   |
```

Comment:
This table appears malformed. It doesn't render in the RTD CI.

Reply:
Thanks, we will adapt.

Reply:
done